src/arrays/programs_involving_data_sequences/medium/q7/DupeDupeDupeDupe.java

```diff
@@ -0,0 +1,43 @@
+package arrays.programs_involving_data_sequences.medium.q7;
+
+/*
+Make a Java program that calls a method that takes an array of words as input. Go through all the words and find the
```

Comment:
We never actually say that they are building a String, so I think we should rewrite this question to be more clear.